server/src/test/scala/org/scassandra/server/actors/ErrorHandlingBehaviors.scala

```diff
@@ -0,0 +1,86 @@
+package org.scassandra.server.actors
+
+import org.scalatest.FunSuite
+import org.scassandra.server.cqlmessages.{ProtocolProvider, Consistency}
+import org.scassandra.server.cqlmessages.response._
+import org.scassandra.server.priming._
+
+trait ErrorHandlingBehaviors extends ProtocolProvider {
+  this: FunSuite =>
```

Comment:
Its a bit confusing, but its a way of saying that any type that has the ErrorHandlingBehaviors trait must also have the FunSuite trait but that ErrorHandlingBehaviors does not necessarily extend FunSuite.